policyengine_us/parameters/gov/states/il/dhs/tanf/income/disregard/rate.yaml

```diff
@@ -0,0 +1,16 @@
+description: Illinois excludes this share of earnings from the Temporary Assistance for Needy Families program countable income, when computing the benefit value.
+
+values:
+  2020-10-01: 0.75
+
+metadata:
+  unit: /1
+  period: month
+  label: Illinois TANF earned income percentage disregard
+  reference:
+    - title: PM 10-02-02-c:First Regular Roll Month
+      href: https://www.dhs.state.il.us/page.aspx?item=15864
+    - title: Ill. Admin. Code tit. 89, § 112.143 - Recognized Employment Expenses (a)
+      href: https://www.law.cornell.edu/regulations/illinois/Ill-Admin-Code-tit-89-SS-112.143
```

Comment:
Lets use this instead:
https://www.law.cornell.edu/regulations/illinois/Ill-Admin-Code-tit-89-SS-112.141